### PR TITLE
Pull request for WAZO-1660-wrong-internal-links

### PR DESCRIPTION
**Pull request by sduthil**
uc-doc: fix installation link
uc-doc: fix broken internal links
uc-doc: handle unexisting anchor

### CODE REVIEW

File: content/uc-doc/installation/install-system.md [hunk 1]
```diff
@@ -4,4 +4,4 @@ title: Installing the System
 
 ## The install procedure has moved
 
-Please refer to the new install documentation here : <https://wazo-platform.org/install>
\ No newline at end of file
+Please refer to the new install documentation here : <https://wazo-platform.org/install/unified-communication>
```
Comment (fredericlepied) [inline content/uc-doc/installation/install-system.md]: it should be removed and the links pointing here should go to /install/unified-communication

Comment (chartrandf) [reply content/uc-doc/installation/install-system.md]: Good point, fixed!

**Comment by sduthil** [comment]
I will fix the remaining links to `wazo-platform.org` and to `install-system` in another PR.

**Comment by sduthil** [comment]
regate

**Pull request merged**